components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java

```diff
@@ -249,6 +262,12 @@ public Response loginAppNamePost(String appName
             errorDTO.setError(IdPClientConstants.Error.INTERNAL_SERVER_ERROR);
             errorDTO.setDescription("Error in login to the uri '" + appName + "'. Error: " + e.getMessage());
             return Response.serverError().entity(errorDTO).build();
+        } catch (ConfigurationException e) {
+            LOG.debug("Error occurred while reading configs from deployment.yaml. " + e.getMessage()) ;
```

Comment:
Need to check isDebugEnabled

Reply:
Fixed in a0108b09b0816017989673d72ee76c8f34cce7f5